plugins/cilium-docker/driver/driver.go

```diff
@@ -399,7 +401,7 @@ func (driver *driver) createEndpoint(w http.ResponseWriter, r *http.Request) {
 		var veth *netlink.Veth
 		veth, _, _, err = connector.SetupVeth(create.EndpointID, int(driver.conf.DeviceMTU),
 			int(driver.conf.GROMaxSize), int(driver.conf.GSOMaxSize),
-			int(driver.conf.GROIPV4MaxSize), int(driver.conf.GSOIPV4MaxSize), endpoint)
+			int(driver.conf.GROIPV4MaxSize), int(driver.conf.GSOIPV4MaxSize), endpoint, sysctlDisable)
```

Comment:
This doesn't compile anymore

Reply:
Sorry about that one. Refactoring can be harder than it looks sometimes :smiling_face_with_tear: